furnace.c

```diff
@@ -25,6 +25,9 @@
 #define STR(X) STR_HELPER(X)
 #define STR_HELPER(X) #X
 
+#define MAGNETRON_PIN       10
+#define MAGNETRON_PULSE_MS  50
+
```

Comment:
Consider moving to `magnetron.c` or guarding with `#if CONFIG_MAGNETRON`, so we know we don't use it when It's disabled.